Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -1063,20 +1056,21 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>
 #endif // PLATFORM(MAC)
 
 #if PLATFORM(GTK)
-    PlatformViewWidget viewWidget();
+    GtkWidget* viewWidget();
     bool makeGLContextCurrent();
+#endif
 
-#if HAVE(APP_ACCENT_COLORS)
+#if PLATFORM(GTK) && HAVE(APP_ACCENT_COLORS)
     void accentColorDidChange();
-#endif
 #endif
 
-    const std::optional<WebCore::Color>& backgroundColor() const { return m_backgroundColor; }
+    const std::optional<WebCore::Color>& backgroundColor() const;
     void setBackgroundColor(const std::optional<WebCore::Color>&);
 
 #if PLATFORM(WIN)
-    PlatformViewWidget viewWidget();
+    HWND viewWidget();
```

Comment:
You need to make this corresponding change in `UIProcess/win/WebPageProxyWin.cpp`